docs/modules/operation/pages/database-reports/templates/avail-summary.adoc

```diff
@@ -1,30 +1,65 @@
 [[avail-summary-report]]
-= Availability Summary Report
+= Availability Summary Report (Default Configuration for Past 7 Days)
 
-Summary of availability across one or more surveillance categories.
+
+== Report Contents
+
+This report provides a summary of availability across one or more surveillance categories.
+One page report with average, minimum and maximum values for the following availability data for the specified surveillance category over the specified date range:
+
+* Number of Outages
+* Mean Time To Restore ("MTTR") (in hours)
```

Comment:
I think this might be better all caps at it is usually written as MTTR (I had to look it up).  Perhaps if we started with the abbreviation and put the spelled out version in parens it might look better?

Reply:
Fine to keep the initial caps.